openmls/src/prelude.rs

```diff
@@ -34,6 +34,7 @@ pub use crate::extensions::*;
 // Framing
 // TODO #265: This should mostly disappear
 pub use crate::framing::{errors::*, sender::Sender, *};
+pub use crate::framing::{MlsMessageIn, MlsMessageOut};
```

Comment:
Is this really needed, given that we already expose `*` in the line above?

Reply:
Apparently yes, if I remove that line it yells at me with error messages like the following.
```
error[E0433]: failed to resolve: use of undeclared type `MlsMessageOut`
   --> src/user.rs:120:13
    |
120 |             MlsMessageOut::Ciphertext(mls_ctxt_ptr) => *mls_ctxt_ptr,
    |             ^^^^^^^^^^^^^ use of undeclared type `MlsMessageOut`
```

Taking a step back, this might be because `MlsMessageIn` is defined in `crate::framing::message` and not `crate::framing`, so an alternative is to instead use
```
use crate::framing::message::*
```

Reply:
Right, I guess it should be

```
pub use crate::framing::{errors::*, message::*, sender::Sender, *};
```